src/modules/fancyzones/editor/FancyZonesEditor/Models/LayoutModel.cs

```diff
@@ -240,7 +277,7 @@ public static ObservableCollection<LayoutModel> LoadCustomModels()
             }
             catch (Exception ex)
             {
-                ShowExceptionMessageBox("Error loading custom layouts", ex);
+                ShowExceptionMessageBox(ErrorLoadingCustomLayouts, ex);

```

Comment:
Shouldn't we localize all strings that we are presenting to a user?

Reply:
You mean in other files as well? Sure, just going step by step. :)